godot-core/src/builtin/vector4.rs

```diff
@@ -4,13 +4,19 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-use std::fmt;
+use std::{
+    fmt,
+    ops::{Index, IndexMut},
+};
```

Comment:
IIRC we don't use this import style elsewhere, maybe keep it flat for now (i.e. two `use` statements).

I think there are nightly rustfmt configs to tweak that, but so far we don't have automation on this 🙂

Reply:
i didn't do this manually so i think this is something rust-analyzer + rustfmt concocted